cleanlab/datalab/issue_manager/label.py

```diff
@@ -118,9 +121,19 @@ def _reset(self) -> None:
 
     def find_issues(
         self,
-        pred_probs: np.ndarray,
+        pred_probs: Optional[npt.NDArray] = None,
+        features: Optional[npt.NDArray] = None,
         **kwargs,
     ) -> None:
+        if pred_probs is None and features is None:
+            raise ValueError("Either pred_probs or features must be provided to find label issues.")
+        if pred_probs is None:
+            # produce out-of-sample pred_probs from features
+            knn = KNeighborsClassifier(n_neighbors=self._k)
+            pred_probs = cross_val_predict(
+                knn, features, self.datalab.labels, cv=5, method="predict_proba"
+            )
```

Comment:
[nit] these checks could be condensed a little nicer:

```suggestion
        if pred_probs is None:
            if features is None:
                raise ValueError("Either pred_probs or features must be provided to find label issues.")
            # produce out-of-sample pred_probs from features
            knn = KNeighborsClassifier(n_neighbors=self._k)
            pred_probs = cross_val_predict(
                knn, features, self.datalab.labels, cv=5, method="predict_proba"
            )
```